nipoppy_cli/tests/test_workflow_init.py

```diff
@@ -33,23 +28,30 @@ def test_run(dpath_root: Path):
     assert Path(dpath_root, FPATH_CONFIG).exists()
     assert Path(dpath_root, FPATH_MANIFEST).exists()
 
-    # check that descriptor files have been copied
-    for fpath in DPATH_DESCRIPTORS.iterdir():
-        assert Path(
-            dpath_root, ATTR_TO_DPATH_MAP["dpath_descriptors"], fpath.name
-        ).exists()
-
-    # check that sample invocation files have been copied
-    for fpath in DPATH_INVOCATIONS.iterdir():
-        assert Path(
-            dpath_root, ATTR_TO_DPATH_MAP["dpath_invocations"], fpath.name
-        ).exists()
-
-    # check that sample tracker config files have been copied
-    for fpath in DPATH_TRACKER_CONFIGS.iterdir():
-        assert Path(
-            dpath_root, ATTR_TO_DPATH_MAP["dpath_tracker_configs"], fpath.name
-        ).exists()
+    # check that pipeline config files have been copied
+    for pipeline_configs in (
+        workflow.config.BIDS_PIPELINES,
+        workflow.config.PROC_PIPELINES,
+    ):
+        for pipeline_config in pipeline_configs:
+            assert (
```

Comment:
Just a nitpick, but this part could be modularized to make the test less verbose.

e.g.

```python3
def exist_or_none(o: object, s: str) -> bool:
  if attr := getattr(o, s, None):
    return attr.exist()
  return True
```